### PR TITLE
docs - add install step for fingerprint verification

### PR DESCRIPTION
**Pull request by lisakowen**
restructure the "install greenplum" topic to include tasks for downloading/obtaining the software and verifying the SHA.  some content has been conditionalized.

doc review site link (behind VPN):  https://docs-lisa-install-verify.sc2-04-pcf1-apps.oc.vmware.com/7-0/install_guide/install_gpdb.html

also attaching a pdf for oss doc changes:
[install_oss.pdf](https://github.com/greenplum-db/gpdb/files/7341747/install_oss.pdf)

### CODE REVIEW

File: gpdb-doc/dita/install_guide/verify_sw.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    from VMware Tanzu Network.</shortdesc>
+  <body>
+    <p>VMware generates a SHA256 fingerprint for each Greenplum Database software
+      download available from <xref href="https://network.pivotal.io/products/pivotal-gpdb"
+        format="html" scope="external">VMware Tanzu Network</xref>. This fingerprint enables
+      you to verify that your downloaded file is unaltered from the original.</p>
+    <p>After you download a Greenplum Database server or component software package,
+      you can verify the integrity of the software as follows:</p>
+    <ol>
+      <li>On the VMware Tanzu Greenplum <i>Release Download Files</i> page, click the
```
Comment (mperezfuster) [inline gpdb-doc/dita/install_guide/verify_sw.xml:16]: maybe this location could be made a bit clearer, it took me a while to realise I had to select the product to see the i icon.

**Review by mperezfuster — APPROVED** [review]
thanks for creating this! I only had a small comment

File: gpdb-doc/dita/install_guide/verify_sw.xml (hunk 1 above)
Comment (lenapivotal) [reply gpdb-doc/dita/install_guide/verify_sw.xml:16]: I agree, we could mention the word "dropdown" somewhere as it's not obvious where the "I" is ....

**Review by lenapivotal — APPROVED** [review]
LGTM

**Comment by lisakowen** [comment]
thanks for reviewing @mperezfuster and @lenapivotal .  i was assuming that the user had just downloaded the package and was already on the appropriate page (Release Download Files).  given that might not be the case, i just added an explicit step to locate the package.

**Comment by dyozie** [comment]
Can you check out the conflict for the install guide file? I was thinking you could use something like the modified text for download install, that covers both open source and proprietary versions.  Also, it would be great if for the new verifying topic if you could remove the condition but add (VMware Tanzu SQL) at the end of the title.

For context, I checked in some dita cleanup prior to converting to markdown which edited some passages to remove use of conditional text in the install guide.

**Comment by lisakowen** [comment]
@dyozie - i fixed the merge conflict (removed conditionals, removed oss-specific topic).  also renamed the verify topic and removed an unrelated condition.  let me know if you were looking for something different.

**Comment by dyozie** [comment]
Thanks @lisakowen - that looks great!

**Pull request merged**